frontend/src/main/components/PersonalSchedules/PersonalSchedulePanel.js

```diff
@@ -0,0 +1,189 @@
+import React from "react";
+import { Container, Row, Col, Card } from "react-bootstrap";
+import SchedulerEvents from "./PersonalScheduleEvent"; // Updated import path
+
+const daysOfWeek = [
+  "Monday",
+  "Tuesday",
+  "Wednesday",
+  "Thursday",
+  "Friday",
+  "Saturday",
+  "Sunday",
+];
+
+const hours = [
+  "1 AM",
+  "2 AM",
+  "3 AM",
+  "4 AM",
+  "5 AM",
+  "6 AM",
+  "7 AM",
+  "8 AM",
+  "9 AM",
+  "10 AM",
+  "11 AM",
+  "12 PM",
+  "1 PM",
+  "2 PM",
+  "3 PM",
+  "4 PM",
+  "5 PM",
+  "6 PM",
+  "7 PM",
+  "8 PM",
+  "9 PM",
+  "10 PM",
+  "11 PM",
+];
```

Comment:
This code is duplicated in both the PersonalSchedulePanel.js component, and it's tests.

Can we instead, move this into a constant exported from timeUtils.js or a new dateUtils.js file and then reference from there in both places?